README.md

```diff
@@ -1,24 +1,168 @@
-# README
+# Food App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
 
-Things you may want to cover:
+# Table of Contents
 
-* Ruby version
+- [About the Project](#about-project)
+  - [Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [Authors](#authors)
+- [Future Features](#future-features)
+- [Contributing](#contributing)
+- [Show your support](#support)
+- [Acknowledgements](#acknowledgements)
+- [FAQ](#faq)
+- [License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# [Food App] <a name="about-project"></a>
 
-* Database creation
+**[Food App]** The Food app will be a classic example of a Food App. It is a functional website that will show the list of shopping and empower readers to interact with them by adding recipes and inventories and generating shopping list. It has been made by the following ERD diagram.
 
-* Database initialization
+## Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a>https://www.ruby-lang.org/en/</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<!-- Features -->
 
-* ...
+### Key Features <a name="key-features"></a>
+
+ - [ ] **[Creating a data model]**
+ - [ ] **[Processing data in models]**
+ - [ ] **[Validations and Model specs]**
+ - [ ] **[Setup and controllers]**
+ - [ ] **[Controllers specs]**
+ - [ ] **[Views]**
+ - [ ] **[Forms]**
+ - [ ] **[Integration specs for Views and fixing n+1 problems]**
+ - [ ] **[Add Devise]**
+ - [ ] **[Add authorization rules]**
+ - [ ] **[Add API endpoints]**
+ - [ ] **[API documentation]**
```

Comment:
in this key features list, there feature that have not been implemented in this project ilke:
**[Controllers specs]**
**[Add API endpoints]**
 **[API documentation]**
so I suggest that this list be revisited and list only the features that have been implemented